tests/cli.rs

```diff
@@ -0,0 +1,25 @@
+use assert_cmd::prelude::*;
+use predicates::prelude::*;
+use std::process::Command;
+
+#[test]
+fn validates_check_file() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
How do you think we should go about adding an integration test for ensuring that TLint correctly invalidates an invalid Check file? Should I create an invalid Check inside `tests/`?

Reply:
Yes please :heart: feel free to move the `check.yml` file under `tests/fixtures` and create a generic invalid check to test against. Just one for the moment will do I think :+1: